src/defaultTokens.ts

```diff
@@ -0,0 +1,31 @@
+// All entries must match the name of a folder under the data/ directory and be upper-cased.
```

Comment:
could consider adding these as a property too like opListId or something

Reply:
yeah I was considering doing that, it may make things easier so that we do not need to actually generate multiple json files

Reply:
I decided to go with the opListId approach